### ISSUE TITLE
sys_time regression on Windows

### ISSUE THREAD

**Issue by Simn** [HaxeFoundation/hashlink#296]
With latest HL:

```haxe
class Main {
	static function main() {
		var t0 = Sys.time();
		var im = [1 => ""];
		for (i in 0...10000000)
			im[1] += "";
		trace(im[1], Sys.time() - t0);
	}
}
```

![powershell_ise_2019-09-05_14-51-43](https://user-images.githubusercontent.com/634365/64344496-e30da880-cfee-11e9-81ec-88c9f31410c7.png)

This gives consistent results on 1.10.

**Comment by Simn** [comment]
Reverting https://github.com/HaxeFoundation/hashlink/commit/a02522b1327c1a5bc3da6c8139ee7d02ade12e2a fixes it

**Comment by ConstNW** [comment]
Are you sure it's code depended?
```
PS D:\devel\hl-pr\285\bin> Measure-Command {hl bin.hl}


Days              : 0
Hours             : 0
Minutes           : 0
Seconds           : 6
Milliseconds      : 189
Ticks             : 61892230
TotalDays         : 7,1634525462963E-05
TotalHours        : 0,00171922861111111
TotalMinutes      : 0,103153716666667
TotalSeconds      : 6,189223
TotalMilliseconds : 6189,223



PS D:\devel\hl-pr\285\bin> hl .\bin.hl
src/Main.hx:19: , 6.23729181289673
```

**Comment by Simn** [comment]
Well, I'm sure that the example takes less than 11 seconds to execute. So the reported time difference is not accurate for me.

**Comment by Simn** [comment]
I don't think it matters, but I'm building HL with Windows SDK 10.0 (latest), platform toolset Visual Studio 2019 (v142) as an x64 release version.

**Comment by ncannasse** [comment]
@Simn could you retry now that we have merged #285 ?

**Comment by Simn** [comment]
Works now!
